src/app/views/jobs/jobs.component.scss

```diff
@@ -48,8 +48,13 @@
 
 .job-list-container {
   background-color: $default-bg-color;
+  min-height: 1026px;
```

Comment:
No it is the same as 10 list items in height. Temporary fix so the list view does not jump when list items are populated.